### PR TITLE
[Mistweaver] update MistweaverHealingEfficiency.js

### PR DESCRIPTION
**Pull request by pingypong**
fixed a few copy/paste errors affecting the overheal column in detailed view

### CODE REVIEW

File: src/parser/monk/mistweaver/modules/features/MistweaverHealingEfficiencyTracker.js [hunk 1]
```diff
@@ -103,23 +104,26 @@ class MistweaverHealingEfficiencyTracker extends HealingEfficiencyTracker {
...
     return spellInfo;
   }
 
   getYulonDetails(spellInfo) {
     // Get all soothing breath and enveloping breath healing since its all bc of yu'lon
     spellInfo.healingDone = this.healingDone.byAbility(SPELLS.SOOTHING_BREATH.id).effective;
-    spellInfo.overhealingDone = this.healingDone.byAbility(SPELLS.SOOTHING_BREATH.id).effective;
-    // Enveloping breath part
+    spellInfo.overhealingDone = this.healingDone.byAbility(SPELLS.SOOTHING_BREATH.id).overheal;
+    // Enveloping breath part -- should this be attributed to yulon? we attribute viv cleave healing to rem but here we don't attribute the envB to envM? will keep for now
```
Comment (abelito75) [inline src/parser/monk/mistweaver/modules/features/MistweaverHealingEfficiencyTracker.js:115]: This makes sense. Put it there

**Review by abelito75 — CHANGES_REQUESTED** [review]
Outside of the envb thing it looks good

**Comment by pingypong** [comment]
care before merging this, looks like there's an issue with mana efficiency caused by another change on shadowland.wowanalyzer.com: 
image

**Comment by Pewtro** [comment]
have you tried to run `rm -rf nodemodules` followed by a `yarn`?

**Comment by pingypong** [comment]
my local is running fine, that SS is from shadowlands.wowanalyzer.com

**Comment by abelito75** [comment]
I should have stated this then leaving it implied but you should go edit MistweaverHealingEfficiencyDetails to state that envb in bundled up with env

**Comment by pingypong** [comment]
I'm not sure what you mean @abelito75 by that, do you a more detailed comment?

**Comment by abelito75** [comment]
MistweaverHealingEfficiencyDetails is where we state that 
`<SpellLink id={SPELLS.GUSTS_OF_MISTS.id} /> healing is added to the appropriate spell that caused the gust. <SpellLink id={SPELLS.ESSENCE_FONT.id} /> is given the healing from duplicated gusts, since without <SpellLink id={SPELLS.ESSENCE_FONT.id} /> the second gust would not have happened. <SpellLink id={SPELLS.RENEWING_MIST.id} /> is given the splash healing of <SpellLink id={SPELLS.VIVIFY.id} />'s heal since without <SpellLink id={SPELLS.RENEWING_MIST.id} />, <SpellLink id={SPELLS.VIVIFY.id} /> wouldn't have splashed`
which means we should also say the same about enveloping mist and enveloping breath

**Comment by pingypong** [comment]
added some breaks in as well since it was becoming quite the wall of text 
image

**Pull request merged**